runtime/js/99_main.js

```diff
@@ -589,6 +590,10 @@ delete Object.prototype.__proto__;
     numCpus = cpuCount;
     registerErrors();
 
+    reportError.setPrintException((jsError) =>
+      core.opSync("op_print_exception", jsError)
+    );
```

Comment:
This could use a comment too

Reply:
This is just a dependency injection similar to the ones strewn above it. Is there something unintuitive about it? If so I would put that above the `setPrintException()` declaration.

Reply:
I think a comment stating that this is hit only for uncaught errors coming from workers would be sufficient (unless that's not true in which case a comment would be even more appreciated :D)

Reply:
It's hit whenever an `"error"` event is dispatched by the runtime and `e.preventDefault()` isn't called. I added a comment above the function.